src/plugins/discover/public/hooks/use_root_breadcrumb.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { useEffect, useState } from 'react';
+import { AggregateQuery, Filter, Query, TimeRange } from '@kbn/es-query';
+import { DiscoverAppLocatorParams } from '../locator';
+import { useDiscoverServices } from './use_discover_services';
+
+export const useRootBreadcrumb = ({
```

Comment:
Yes this makes much sense to create a shared hook 👍 . 2 asks here
Could you add units tests and add a high level description what this hook does and where it is used? many thx!

Reply:
Updated.